en/developer_guide/entity_list_ajax_pagination.rst

```diff
@@ -200,15 +204,72 @@ For instance, the ``Redirection`` entity will look like:
         // ...
     }
 
-``type`` will take ``string`` as a default parameter.
-``name`` will take the name of the property if not set.
+Yaml:
+
+.. code-block:: yaml
+
+    # src/AppBundle/Resources/config/search/Redirection.yml
+
+    AppBundle\Document\AppBundle\Document:
+        properties:
+            locale:
+                key: locale
+            siteName:
+                key: site_name
+            routePattern:
+                key: route_pattern
+            url:
+                key: [redirection, url]
+            permanent:
+                key: permanent
+                type: boolean
+
+Xml:
+
+.. code-block:: xml
+
+    <!-- src/AppBundle/Resources/config/search/Redirection.xml -->
+
+    <search-mapping>
+        <class name="AppBundle\Document\AppBundle\Document">
+            <field field="locale" key="locale" />
+            <field field="siteName" key="site_name" />
+            <field field="routePattern" key="route_pattern" />
+            <field field="url" key="redirection, url" />
+            <field field="locale" key="locale" />
+            <field field="permanent" key="permanent" type="boolean"/>
+            <field field="permanent" key="fake_property2, fake_property_multi"/>
+        </class>
+    </search-mapping>
 
-The ``open_orchestra_api.annotation_search_reader`` will build the ``descriptionEntity``.
+The ``open_orchestra_api.annotation_search_reader`` will extract the ``descriptionEntity``.
 
 .. code-block:: php
 
     $mapping = $this->get('open_orchestra_api.annotation_search_reader')
         ->extractMapping('OpenOrchestra\ModelBundle\Document\Redirection');
 
+-------
+ Notes
+-------
+
+In mapping :
+
+* ``type`` will take ``string`` as a default parameter if it is not specified.
+* With the mapping in annotation ``name`` will take the name of the property if it is not set.
+
+By default the mapping in xml or yaml should be in folder ``Ressources/config/search`` of your bundle.
```

Comment:
"By default the xml and yaml mapping files should be located in the folder 'xxx' of you bundle"